packages/api/src/resolvers/type/user-types.ts

```diff
@@ -221,4 +222,7 @@ export const User: UserHelperTypes = {
       userId: user._id,
     });
   },
+  webPushEnabled: async (_, __, { modules, userId, userAgent }) => {
+    return modules.users.pushEnabledByUser({ userId, userAgent });
```

Comment:
See my other comments: This does return true on 3 devices when using same hardware and same browser for the same user even when only 1 device is subscribed.